structure/0006-governance.md

```diff
@@ -2,53 +2,135 @@
+
+After one week of preparation and nomination submission, nominations must close, the list of candidates must be published, and the vote must begin, lasting one week. The system used to conduct the vote must allow voters to change their votes, and enable voters to rank the candidates in order of preference, while also having the option to not rank any given candidate. All organization members (which at this time is anyone with the "Community Team" or "Quilt Developer" role in Discord) are able to vote. During the vote, voters can contact the EOB about any concerns they have over the vote (for example, concerns about a candidate being elected, or concerns about the vote being influenced by an external force), to be dealt with as outlined in the paragraph below.
+
+If the EOB receives any concerns over a candidate, either internally or submitted by voter, they must decide if the concerns are worth investigating. If they are not, no further action is taken. If they are, the team must investigate, and, if significant results are found, must publish the results to the voters, so that they can factor them into their decision. The EOB cannot prevent any candidate from running, this is to ensure that they don't have direct control over who can be elected. If, however, the vote is due to end before the EOB can complete their investigation, they may delay the end of the vote by up to 30 additional days.
+
+After the voting period concludes, the EOB may take up to one week to count the votes. In the event of a tie, votes by existing admin board members will count as three votes. The EOB may privately inform the winning candidate, before publishing the results to the organization members and subsequently the community at large.
+
+## Processes
+
+To streamline working with teams, some standard processes need to be defined.
+The following processes apply to all teams unless that team's RFC specifies a different process.
+
+### Voting
+
+Unless specified otherwise, the following process is used for voting:
+
+A vote in a team may be called at any time by a team lead for that team or for one of its parent teams.
+A member of the administrative board may also initiate a vote in any team. 
+
+When a vote is called within a team, all members of that team, its parent teams, and the administrative board are eligible to vote.
+All polls should include a positive and negative option, as well as the option to abstain.
+Voting members that abstain are excluded from percentages and tallies.
+
+After one week, the plurality of votes wins.
+
+### Removal of Staff
+
+Any team member or administrative board member may be removed by a majority vote of all eligible organization members.
+The following members are eligible:
+- All members of the relevant team
```

Comment:
Should we make frequent contributors to the team's GitHub also eligible? A situation like EnnuiL before she actually *joined* the QFAPI team comes to mind.

Reply:
Aren't those people also part of our community?